### ISSUE TITLE
unhandled error

### ISSUE THREAD

**Issue by AlvaroVega** [telefonicaid/fiware-pep-steelskin#256]
```
time=2015-08-25T23:39:24.796Z | lvl=ERROR | corr=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | trans=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | op=/v1/updateContext | msg=Authentication rejected: the token does not match the service
time=2015-08-25T23:39:24.796Z | lvl=ERROR | corr=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | trans=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | op=/v1/updateContext | msg=VALIDATION-GEN-003] Error connecting to Keystone authentication: TOKEN_DOES_NOT_MATCH_SERVICE: The provided token does not belong to the provided service.
time=2015-08-25T23:39:24.796Z | lvl=ERROR | corr=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | trans=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | op=/v1/updateContext | msg=Authentication rejected: the token does not match the service
time=2015-08-25T23:39:24.796Z | lvl=ERROR | corr=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | trans=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | op=/v1/updateContext | msg=VALIDATION-GEN-003] Error connecting to Keystone authentication: TOKEN_DOES_NOT_MATCH_SERVICE: The provided token does not belong to the provided service.
time=2015-08-25T23:39:24.797Z | lvl=ERROR | corr=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | trans=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | op=/v1/updateContext | msg=Authentication rejected: the token does not match the service
time=2015-08-25T23:39:24.797Z | lvl=ERROR | corr=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | trans=0bae6580-8a8a-44c8-9d6c-02b26de572c4 | op=/v1/updateContext | msg=VALIDATION-GEN-003] Error connecting to Keystone authentication: TOKEN_DOES_NOT_MATCH_SERVICE: The provided token does not belong to the provided service.
time=2015-08-25T23:39:24.802Z | lvl=ERROR | corr=57ce04b5-7603-4d68-9aa0-10fb4cb65214 | trans=57ce04b5-7603-4d68-9aa0-10fb4cb65214 | op=/v1/updateContext | msg=Error: Can't set headers after they are sent.
time=2015-08-25T23:39:24.817Z | lvl=ERROR | corr=7c1c1b9e-0460-41c0-85b3-5140ef37ca4d | trans=7c1c1b9e-0460-41c0-85b3-5140ef37ca4d | op=/v1/updateContext | msg=Error: Can't set headers after they are sent.

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (http.js:690:11)
    at ServerResponse.res.setHeader (/opt/pepProxy/node_modules/express/node_modules/connect/lib/patch.js:63:22)
    at next (/opt/pepProxy/node_modules/express/node_modules/connect/lib/proto.js:156:13)
    at retry (/opt/pepProxy/lib/services/keystoneAuth.js:502:13)
    at /opt/pepProxy/node_modules/async/lib/async.js:254:17
    at /opt/pepProxy/node_modules/async/lib/async.js:157:25
    at /opt/pepProxy/node_modules/async/lib/async.js:251:21
    at EventEmitter.<anonymous> (/opt/pepProxy/node_modules/async/lib/async.js:615:34)
    at EventEmitter.emit (events.js:95:17)
    at Request._callback (/opt/pepProxy/lib/services/keystoneAuth.js:454:33)
Changing PROXY_PORT to environment value: 10026
Changing TARGET_HOST to environment value: 127.0.0.1
Changing TARGET_PORT to environment value: 1026
Changing LOG_LEVEL to environment value: ERROR
Changing ACCESS_HOST to environment value: 10.0.0.2
Changing ACCESS_PORT to environment value: 8080
Changing ADMIN_PORT to environment value: 11211
Changing AUTHENTICATION_HOST to environment value: 10.0.0.2
Changing AUTHENTICATION_PORT to environment value: 5001
Changing PROXY_USERNAME to environment value: pep
Changing PROXY_PASSWORD to environment value: 6deDuURN
Changing COMPONENT_PLUGIN to environment value: orion
Loading middlewares
Server started
time=2015-08-25T23:39:37.326Z | lvl=ERROR | corr=6eefc6e4-3f25-4326-a384-8c978c5cd23a | trans=6eefc6e4-3f25-4326-a384-8c978c5cd23a | op=/v1/queryContext | msg=Invalid PEP token: null
```

**Comment by fgalan** [comment]
Fixed in PR #257 (release/0.7.0). Fix in develop is still pending.

**Comment by fgalan** [comment]
Fixed in PR #258 (develop)
